src/queries.js

```diff
@@ -1,12 +1,26 @@
 import CozyClient, { Q } from 'cozy-client'
 
 export const defaultFetchPolicy = CozyClient.fetchPolicies.olderThan(30 * 1000)
+
 export const appsConn = {
   query: Q('io.cozy.apps'),
   as: 'io.cozy.apps',
   fetchPolicy: defaultFetchPolicy
 }
 
+export const instanceSettingsConn = {
+  query: Q('io.cozy.settings').getById('io.cozy.settings.instance'),
+  as: 'io.cozy.settings/instance_standalone',
+  fetchPolicy: defaultFetchPolicy,
+  singleDocData: true
+}
+
+export const homeSettingsConn = {
+  query: Q('io.cozy.home.settings').limitBy(1),
+  as: 'io.cozy.home.settings',
+  fetchPolicy: defaultFetchPolicy
+}
```

Comment:
Really a nit, don't need to change it : since you're doing limitBy(1), you could have used `singleDocData: true` in order to get response.data instead of response.data[0]